rtirq.sh

```diff
@@ -356,13 +356,27 @@ function rtirq_reset ()
 	rtirq_high reset
 }
 
+#
+# Warn about prerequisites to script usefulness
+#
+function rtirq_check_sanity ()
+{
+	if [[ "$(uname -v)" == *"PREEMPT "* ]]; then
```

Comment:
shouldn't it be `*"PREEMPT_RT"*` instead?

Reply:
Yes! I'll fix it.

Reply:
Done!